src/translations/en/case-studies.mdx

```diff
@@ -1,188 +1,49 @@
-
-<LinkIcon
-  iconColor="var(--colorBlack400)"
-  href="https://github.com/ambanum/OpenTermsArchive-versions/commit/0d2f0386"
-  small={true}
-  className="mb__L"
->
-  <a target="_blank" rel="noopener">
-    See the changes
-  </a>
-</LinkIcon>
+On November 2, 2021, Qwant <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/549e959ef7671a194b9bedba8d12c8031c39b922">specifies</a> in its privacy policy, the type of data it passes on to its partner Microsoft "for the security and reliability of its services." Among the data that the service can share with Bing, the search engine of the American giant, are the keywords searched, the type and version of the browser used, or an identifier calculated from the IP address and browser information.
 
 ---
 
-### Pinterest
+### Instagram blocks data retrieval and clarifies what information it retrieves
 
-#### September 3, 2020
+In November 2021, Instagram made several updates to its <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/9998cb21b69a222540491a81dd05be19a9785891">community rules</a>, its <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/9aaf5dbfc3b34d45d640b0c7f96dce3598c7bcd2">terms of use</a> and its <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/927ceba3865d79cd1b123d9f4b9928b8e997070c">privacy policy</a>. Among the notable developments, the platform explicitly prohibits the sale or exchange of "non-medical or pharmaceutical substances." It says it will remove all posts promoting such products, either for sale or to indicate personal consumption, except in a healing context.
 
-\*_Pinterest_ removed their mentions of the Privacy Shield in their terms of service. In a ruling handed down on Thursday, July 16, 2020 (Schrems II ruling), the EU’s top court ruled that this key data-sharing mechanism was invalid. The European Court of Justice more specifically said that the EU-US Privacy Shield failed to protect privacy and data protection rules. The decision has had immediate and complex implications for data sharing between the EU and US. See the changes here and there.
+In its updated terms of service, Instagram prohibits creating accounts or using them in such a way as to retrieve information without permission, especially in an automated way. This could have an effect on the work of researchers like those of the NGO Algorithm Watch: a research project of the latter had already been closed by Instagram <a target="_blank" rel="noopener" href="https://algorithmwatch.org/en/instagram-research-shut-down-by-facebook/">by summer 2021</a>. The platform also specifies the prohibition of disseminating another user's personal information, including data or publications that are the intellectual property of another Internet user.
```

Comment:
```suggestion
In its updated terms of service, Instagram prohibits creating accounts or using them in such a way as to retrieve information without permission, especially in an automated way. This could have an effect on the work of researchers like those of the NGO Algorithm Watch, a research project of which having already been <a target="_blank" rel="noopener" href="https://algorithmwatch.org/en/instagram-research-shut-down-by-facebook/">closed by Instagram</a> in the summer of 2021. The platform also prohibits disseminating other users’ personal information.
```

Reply:
done